client/src/components/ApplicantProfile.js

```diff
@@ -49,15 +69,15 @@ const ApplicantProfile = () => {
   ];
 
   const createProfile = () => {
-    const formData = { ...profileData, experiences: experienceData };
+    const formData = { ...profileData, experiences: experiences };
     const token = localStorage.getItem("token");
     newProfile(formData, token).then(isSuccessful => {
       setProfileCreated(isSuccessful);
     });
   };
 
   return (
-    <Form onSubmit={createProfile}>
+    <Form.Group onSubmit={createProfile}>
```

Comment:
To make `Form.Group` you have to wrappe it with `Form` component.
E.g 
```
 <Form>
      <Form.Group onSubmit={createProfile}>
        <Form.Field>
          <label htmlFor="job_title">Job title</label>
          <input
            id="job_title"
            name="job_title"
            placeholder="Job title"
            onChange={handleChange}
          />
        </Form.Field>
      </Form.Group>
 </Form>
```

Reply:
I think we agreed to revert this change. the ApplicantProfile component should remain a Form. The Experience should be a Form.Group

Reply:
Do I have to wrap the Form.Group in the Experience component into a Form element or keep it as it is in the Form.Group element ?